compose/config/config.py

```diff
@@ -478,7 +475,7 @@ def process_config_file(config_file, environment, service_name=None):
         'service',
         environment)
 
-    if config_file.version in (V2_0, V2_1, V3_0, V3_1):
+    if config_file.version > V1:
```

Comment:
It works for now, but yes I could see it breaking later, especially if we add patch versions. A real `Version` object would be a lot safer. I'll use ` != V1` for now.